### DISCUSSION TITLE
Welcome to malheatmap Discussions!

### DISCUSSION THREAD

**Discussion by andersonkrs** [andersonkrs/malheatmap#647] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SuryanshChopra** [comment]
Hey, Just wondering if you can add support to Dark Theme on MAL Heatmap, similar to like below example

![image](https://github.com/user-attachments/assets/d075350b-1377-496e-ba01-93777d889a5d)

The above result was obtained by third party chrome extensions, but I would really like if you can add support to Dark Theme natively.
